### PR TITLE
docs: Add Gen AI doc to User Guide (#275)

### PR DESCRIPTION
**Pull request by Oleksandr123234**
# Pull Request Template

## Description

This change add the "Add Gen AI" page to documentation.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement

## How Has This Been Tested?

Docusaurus works properly with new documentations. No errors or warnings are generated.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Pull Request contain one commit. I squash my commits.

### CODE REVIEW

File: docs/user-guide/add-ai-assistant.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Add Gen AI Assistant
+
+KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) to help you observe the overall platform status, as well as troubleshoot some issues related to the platform. Assistant's configuration in performed via the Configuration tab, in the Gen AI section.
```
Comment (NikolayMarusenko) [inline docs/user-guide/add-ai-assistant.md:3]: We have two options: First, add a chat assistant to provide platform-related information and assist with any questions. Second, enable integration with CodeMie, which will onboard the application into both the platform and CodeMie if the checkbox is selected.

File: docs/user-guide/add-ai-assistant.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+As soon as the integration procedure is completed, click the AI chat button at the bottom-right corner of the screen:
+
+  ![AI chat window](../assets/user-guide/ai-chat-button.png "AI chat window")
+
+Type something in the chat to make sure the assistant works properly:
+
+  ![Ask assistant](../assets/user-guide/ask-assistant.png "Ask assistant")
+
+Besides, you will be able to select the **Integrate with CodeMie** option. This option enables Gen AI capabilities powered by the CodeMie tool for codebases:
```
Comment (NikolayMarusenko) [inline docs/user-guide/add-ai-assistant.md:46]: This is a separate integration. I don't see any description of it in the documentation

File: docs/user-guide/add-ai-assistant.md (hunk 1 above)
Comment (Oleksandr123234) [reply docs/user-guide/add-ai-assistant.md]: Rephrased

File: docs/user-guide/add-ai-assistant.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Add Gen AI Assistant
+
+The KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) to help you by providing platform-related information and answering any questions. Assistant's configuration is performed via the Configuration tab, in the Gen AI section.
```
Comment (NikolayMarusenko) [inline docs/user-guide/add-ai-assistant.md:3]: The KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) it is not true

**Review by NikolayMarusenko — APPROVED** [review]

**Pull request merged**